biothings/hub/dataindex/snapshot_repo.py

```diff
@@ -25,6 +25,9 @@ def create(self, **body):
     def delete(self):
         self.client.snapshot.delete_repository(self.name)
 
+    def verify(self):
+        return self.client.snapshot.verify_repository(self.name)
+
```

Comment:
@sengineer0 here using `verify_repository`, it only validates if a repository is working, but we should check if the repo settings match with our config values (e.g. bucket name).

Reply:
It's updated. Please have a check @newgene